augur/api/metrics/contributor.py

```diff
@@ -43,7 +44,7 @@ def contributors(repo_group_id, repo_id=None, period='day', begin_date=None, end
 
     if repo_id:
         contributorsSQL = s.sql.text("""
-           SELECT id::text                         AS user_id,
```

Comment:
If I was casting these are text its probably because the frontend relies on it... checking.

Reply:
I confirmed that we get an issue if we don't cast the result as text: 
```
2023-01-21 16:34:20 linda augur.api.server[200488] ERROR Exception on /api/unstable/repos/61187/contributors [GET]
Traceback (most recent call last):
  File "/home/sean/github/virtualenv/k12/lib/python3.8/site-packages/flask/app.py", line 2073, in wsgi_app
    response = self.full_dispatch_request()
  File "/home/sean/github/virtualenv/k12/lib/python3.8/site-packages/flask/app.py", line 1518, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/home/sean/github/virtualenv/k12/lib/python3.8/site-packages/flask_cors/extension.py", line 165, in wrapped_function
    return cors_after_request(app.make_response(f(*args, **kwargs)))
  File "/home/sean/github/virtualenv/k12/lib/python3.8/site-packages/flask/app.py", line 1516, in full_dispatch_request
    rv = self.dispatch_request()
  File "/home/sean/github/virtualenv/k12/lib/python3.8/site-packages/flask/app.py", line 1502, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**req.view_args)
  File "/home/sean/github/rh-augur-new-dev/augur/api/server.py", line 361, in endpoint_function
    data =  self.transform(func, args, kwargs)
  File "/home/sean/github/rh-augur-new-dev/augur/api/server.py", line 300, in transform
    result = data.to_json(orient=orient, date_format='iso', date_unit='ms')
  File "/home/sean/github/virtualenv/k12/lib/python3.8/site-packages/pandas/core/generic.py", line 2558, in to_json
    return json.to_json(
  File "/home/sean/github/virtualenv/k12/lib/python3.8/site-packages/pandas/io/json/_json.py", line 113, in to_json
    s = writer(
  File "/home/sean/github/virtualenv/k12/lib/python3.8/site-packages/pandas/io/json/_json.py", line 175, in write
    return dumps(
OverflowError: Unsupported UTF-8 sequence length when encoding string
```

Reply:
Weird, I don't know anything about that. I'm assuming that this branch is just old and the old changes wrote over the dev changes. I can make that change really quick though